homeassistant/components/prometheus/__init__.py

```diff
@@ -277,6 +277,15 @@ def _handle_input_boolean(self, state):
         value = self.state_as_number(state)
         metric.labels(**self._labels(state)).set(value)
 
+    def _handle_input_number(self, state):
+        metric = self._metric(
+            "input_number_state",
```

Comment:
Does this needs to be defined as constant?

Reply:
No, I don't think so. It's only used once here, and all other metric names are defined inline.